config/kbin_routes/magazine.yaml

```diff
@@ -13,6 +13,16 @@ magazine_purge:
   path: /m/{name}/purge
   methods: [ POST ]
 
+magazine_purge_content:
```

Comment:
These are the two new paths (the rest is auto formatting to the new `.editconfig` standard)